src/components/Upload/Cropper/EasyCrop.tsx

```diff
@@ -218,14 +247,26 @@ const EasyCrop = forwardRef<EasyCropHandle, EasyCropProps>((props, ref) => {
               iconProps={{
                 path: IconName.mdiRotateRight,
               }}
-              onClick={() => setRotateVal(rotateVal + ROTATE_STEP)}
+              onClick={() => {
+                setButtonType('right');
+                setRotateVal(rotateVal + ROTATE_STEP);
+              }}
               shape={ButtonShape.Round}
               size={ButtonSize.Small}
               theme={mergedTheme}
               themeContainerId={themeContainerId}
               variant={ButtonVariant.SystemUI}
             />
           </Stack>
+          <div
+            className={styles.visuallyhidden}
+            aria-live="assertive"
+            role="status"
+          >
+            {buttonType === 'right'
+              ? `${rotateRightButtonAriaLabelText} ${rotateVal}`
+              : `${rotateLeftButtonAriaLabelText} ${rotateVal}`}
```

Comment:
same as above, we don't want this to update and announce when zoom is pushed. can do a blank string if buttonType is in/out